enjoyhint.js

```diff
@@ -1041,6 +1064,13 @@
                 id: "enjoyhint_arrpw_line"
               })
             );
+
+            $("#poliline").css("stroke", "rgb(255, 255, 255)");
```

Comment:
In some cases set default color can be invoked twice: first time on 1068 line, and the second time in setNarkerMethod.
This part of code can be moved to else part of if statement below.